src/rust/engine/tar_api/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "tar_api"
+version = "0.0.1"
+authors = [ "Pants Build <pantsbuild@gmail.com>" ]
+
+[lib]
+path = "src/tar_api.rs"
+
+[[bin]]
+name = "tar-cli"
+path = "src/main.rs"
+
+[dependencies]
+flate2 = "1.0"
+tar =  "0.4.20"
+clap = "2"
```

Comment:
I think this will add a clap dependency to the engine, which we currently don't have (and which adds a non-trivial compile overhead)

If you want the binary, can you have it as a separate crate? (i.e. make `src/rust/engine/tar_api/tar_cmd` with its own `Cargo.toml` file, and move the binary related stuff into it)

Reply:
Removed the bin.